### PR TITLE
RHV and OSP install code blocks

### PR DESCRIPTION
**Pull request by kalexand-rh**

### CODE REVIEW

**Comment by maxwelldb** [comment]
In my queue. :clown_face:

File: modules/installation-osp-kuryr-octavia-configuration.adoc [hunk 1]
```diff
@@ -44,6 +45,7 @@ the registry. For example:
 . Verify that the `local_registry_images.yaml` file contains the Octavia images.
 For example:
 +
+[source,yaml]
 ----
 ...
```
Comment (maxwelldb) [inline modules/installation-osp-kuryr-octavia-configuration.adoc:50]: Our stylesheet seems to underline the ellipses here. Is that intended?

File: modules/installation-osp-creating-network-resources.adoc [hunk 2]
```diff
@@ -669,18 +669,21 @@ all:
...
 
 . On a command line, create a network, subnet, and router by running the second numbered playbook:
 +
+[source,terminal]
 ----
 $ ansible-playbook -i inventory.yaml 02_network.yaml
 ----
 
-. _Optional_: If you want to control the default resolvers that Nova servers use, run the {rh-openstack} CLI command:
+. Optional: If you want to control the default resolvers that Nova servers use, run the {rh-openstack} CLI command:
```
Comment (maxwelldb) [inline modules/installation-osp-creating-network-resources.adoc:684]: :O

File: modules/installation-osp-deleting-bootstrap-resources.adoc [hunk 3]
```diff
@@ -45,11 +45,12 @@ Delete the bootstrap resources that you no longer need.
...
 ----
 $ ansible-playbook -i inventory.yaml down-03_bootstrap.yaml
 ----
 
 The bootstrap port, server, and floating IP address are deleted.
 
 [WARNING]
-If you have not disabled the bootstrap Ignition file URL, do so now.
\ No newline at end of file
+If you have not disabled the bootstrap Ignition file URL, do so now.
```
Comment (maxwelldb) [inline modules/installation-osp-deleting-bootstrap-resources.adoc:56]: If you did not disable the bootstrap Ignition file URL earlier, do so now. 

?

**Review by maxwelldb — APPROVED** [review]
Saw nothing scary in changes or preview. Made a few nothing comments.

File: modules/installation-osp-kuryr-octavia-configuration.adoc (hunk 1 above)
Comment (kalexand-rh) [reply modules/installation-osp-kuryr-octavia-configuration.adoc:50]: > Our stylesheet seems to underline the ellipses here. Is that intended?

I am not sure. @jboxman, is this something that you've considered in the CSS changes?

Comment (jboxman) [reply modules/installation-osp-kuryr-octavia-configuration.adoc:50]: @maxwelldb @kalexand-rh that's part of the style as written; I've noticed that but cannot say why the author thought it made sense to do so. I can certainly remove it in a future update to the CSS.

**Pull request merged**